docker-compose.ci.yml

```diff
@@ -41,6 +41,7 @@ services:
         volumes:
             - ./config/container.php:/srv/annotations/config.php
             - logs:/srv/annotations/var/logs
+            - .php_cs.cache:/srv/annotations/.php_cs.cache
```

Comment:
so, this is a path that is local to the Jenkins workspace (local clone) so it may be kept around between builds of the same branch but not necessarily across pull requests (if that's needed). Per-project absolute path could be better?

Reply:
also, the `ci` container is the one used to run project tests (which includes `proofreader`/`php-cs-fixer`) so it should be mounted there